dotnet/test/FlexiblePolylineEncoder.Tests/FlexiblePolylineEncoderTests.cs

```diff
@@ -0,0 +1,339 @@
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Text.RegularExpressions;
+using HERE.FlexiblePolyline;
+using NUnit.Framework;
+
+namespace FlexiblePolylineEncoder.Tests
+{
+    public class FlexiblePolylineEncoderTests
+    {
+        [Test]
+        public void EncodedFlexilineMatchesDecodedResultTest()
+        {
+            using (var decodedEnumerator = File.ReadLines("../../../../../../test/round_half_up/decoded.txt").GetEnumerator())
```

Comment:
This path looks wrong, it leaves the source folder altogether. Are you sure you're not referring to a file local to your own computer here?